server/helpers/activityHelper.js

```diff
@@ -0,0 +1,29 @@
+var Yelp = require('yelp');
+
+
+var yelpQuery = new Yelp ({
+  consumer_key: 'AokJ1I73m9ZXRKsQwAFk3w',
+  consumer_secret: 'RtuBaAZctDIRcvTKWBphtGeXVG0',
+  token: 'L5Q9dkCBNNr-GwZDl81mliC8z5OtWJjr',
+  token_secret: '5UIuf3CH89z_ODFJsfQA5RDoFLQ'
```

Comment:
Can you add these to the google doc?
You can add it to the .env file like this (Note that there's no 'var' and no semi-colon.): 
```
.env file

consumer_key = 'consumer key here'
```
Then change it to:
``` 
activityHelper file

consumer_key: process.env.consumer_key,
consumer_secret: process.env.consumer_secret,
token: process.env.token,
token_secret: process.env.token_secret
```